### PR TITLE
Update controller service proto with creating cluster design modifica…

### PR DESCRIPTION
**Pull request by shubh90**
…tions

### CODE REVIEW

File: proto/controller.proto [hunk 1]
```diff
@@ -24,14 +24,87 @@ service TopologyManager {
...
+}
+
+// Request message to delete a cluster.
+message DeleteClusterRequest {
+  string name = 1;
+}
+
+// Returns delete cluster response.
+message DeleteClusterResponse {
+  string response = 1;
```
Comment (alexmasi) [inline proto/controller.proto:89]: what is this field for? same with the deletetoporesponse message

File: proto/controller.proto [hunk 2]
```diff
@@ -24,14 +24,87 @@ service TopologyManager {
   rpc CreateTopology (CreateTopologyRequest) returns(CreateTopologyResponse) {}
   rpc DeleteTopology (DeleteTopologyRequest) returns(DeleteTopologyResponse) {}
   rpc ShowTopology (ShowTopologyRequest) returns(ShowTopologyResponse) {}
+  rpc CreateCluster (CreateClusterRequest) returns(CreateClusterResponse) {}
```
Comment (alexmasi) [inline proto/controller.proto:27]: All of these RPCs should have a doc string one liner

File: proto/controller.proto [hunk 3]
```diff
@@ -42,7 +115,7 @@ message CreateTopologyRequest {
 // Returns create topology response.
 message CreateTopologyResponse {
   string topology_name = 1;
-  NodeState state = 2;
+  TopologyState state = 2;
```
Comment (alexmasi) [inline proto/controller.proto:118]: we may have to add another field here for the kubecfg (so the topo knows which cluster to create in)

File: proto/controller.proto (hunk 2 above)
Comment (shubh90) [reply proto/controller.proto:31]: Done

File: proto/controller.proto (hunk 1 above)
Comment (shubh90) [reply proto/controller.proto]: Made empty response

File: proto/controller.proto (hunk 3 above)
Comment (shubh90) [reply proto/controller.proto:124]: Done

**Review by alexmasi — APPROVED** [review]

**Pull request merged**